_posts/2025-01-09-struct-decode-intro.md

```diff
@@ -0,0 +1,175 @@
e:#333 !important;stroke-width:2px !important" x="-58.5" y="-28.6015625" width="117" height="57.203125"></rect><g class="label" style="" transform="translate(-28.5, -13.6015625)"><rect></rect><foreignObject width="57" height="27.203125"><div xmlns="http://www.w3.org/1999/xhtml" style="display: table-cell; white-space: nowrap; line-height: 1.5; max-width: 200px; text-align: center;"><span class="nodeLabel "><p>Schema</p></span></div></foreignObject></g></g><g class="node default  " id="flowchart-B-14" transform="translate(388, 235.60546875)"><rect class="basic label-container" style="fill:#e1f3f8 !important;stroke:#333 !important;stroke-width:2px !important" x="-77.5" y="-28.6015625" width="155" height="57.203125"></rect><g class="label" style="" transform="translate(-47.5, -13.6015625)"><rect></rect><foreignObject width="95" height="27.203125"><div xmlns="http://www.w3.org/1999/xhtml" style="display: table-cell; white-space: nowrap; line-height: 1.5; max-width: 200px; text-align: center;"><span class="nodeLabel "><p>Logit Bias</p></span></div></foreignObject></g></g><g class="node default  " id="flowchart-C-15" transform="translate(388, 71.6015625)"><rect class="basic label-container" style="fill:#f8e1e1 !important;stroke:#333 !important;stroke-width:2px !important" x="-96.5" y="-28.6015625" width="193" height="57.203125"></rect><g class="label" style="" transform="translate(-66.5, -13.6015625)"><rect></rect><foreignObject width="133" height="27.203125"><div xmlns="http://www.w3.org/1999/xhtml" style="display: table-cell; white-space: nowrap; line-height: 1.5; max-width: 200px; text-align: center;"><span class="nodeLabel "><p>Language Model</p></span></div></foreignObject></g></g><g class="node default  " id="flowchart-D-16" transform="translate(855.5, 88.40234375)"><rect class="basic label-container" style="fill:#f8e1e1 !important;stroke:#333 !important;stroke-width:2px !important" x="-125" y="-28.6015625" width="250" height="57.203125"></rect><g class="label" style="" transform="translate(-95, -13.6015625)"><rect></rect><foreignObject width="190" height="27.203125"><div xmlns="http://www.w3.org/1999/xhtml" style="display: table-cell; white-space: nowrap; line-height: 1.5; max-width: 200px; text-align: center;"><span class="nodeLabel "><p>Constrained Sampling</p></span></div></foreignObject></g></g><g class="node default  " id="flowchart-E-20" transform="translate(1166.25, 88.40234375)"><rect class="basic label-container" style="fill:#e1f8e1 !important;stroke:#333 !important;stroke-width:2px !important" x="-110.75" y="-28.6015625" width="221.5" height="57.203125"></rect><g class="label" style="" transform="translate(-80.75, -13.6015625)"><rect></rect><foreignObject width="161.5" height="27.203125"><div xmlns="http://www.w3.org/1999/xhtml" style="display: table-cell; white-space: nowrap; line-height: 1.5; max-width: 200px; text-align: center;"><span class="nodeLabel "><p>Structured Output</p></span></div></foreignObject></g></g></g></g></g></svg>
```

Comment:
move this to a separate file?

Reply:
If I move it to a separate file it seems not to render correctly.